pwndbg/heap/ptmalloc.py

```diff
@@ -32,6 +32,83 @@ def heap_for_ptr(ptr):
     return ptr & ~(HEAP_MAX_SIZE - 1)
 
 
+class Chunk:
```

Comment:
I really like the idea of modularity for the sake of testing, but I feel like heap-related testing would benefit from further discussion first. A chunk outside the context of GLIBC malloc doesn't make sense e.g. we're expecting a pwndbg command to print a chunk as being free, but to determine that it might first have to locate the heap & arena for that chunk, then iterate through a specific bin.

If you're comfortable with it, we could postpone this until we've discussed heap command testing some more.